src/database/index.tsx

```diff
@@ -217,13 +219,165 @@ export const saveScenario = async (scenario: {}): Promise<void> => {
 
     if (!baselineScenarioRef) return;
 
-    const payload = Object.assign({}, scenario, {
-      updatedAt: currrentTimestamp(),
-    });
+    const payload = buildUpdatePayload(scenario);
 
     baselineScenarioRef.update(payload);
   } catch (error) {
     console.error("Encountered an error while saving the scenario:");
     console.error(error);
   }
 };
+
+export const getFacilities = async (): Promise<Array<Facility> | null> => {
+  try {
+    // Cheating for launch expediency.
+    // See: https://github.com/Recidiviz/covid19-dashboard/issues/129
+    const baselineScenarioRef = await getBaselineScenarioRef();
+
+    if (!baselineScenarioRef) return null;
+
+    const facilitiesResults = await baselineScenarioRef
+      .collection(facilitiesCollectionId)
+      .orderBy("name")
+      .get();
+
+    const facilities = facilitiesResults.docs.map((doc) => {
+      const facility = doc.data() as Facility;
+      facility.id = doc.id;
```

Comment:
Also tacking on the `docRef` id so that we can use it later for updates and deletions